app/upgrades/forks/forks.go

```diff
@@ -20,7 +20,7 @@ func runForkLogicSwapDisable(ctx sdk.Context, keppers *keepers.AppKeepers, _ *mo
 
 		// Disable IBC Channels
 		channelIDs := []string{
-			"channel-1",  // Osmosis
+			"channel-1",  // terra
```

Comment:
please revert this back from "terra" to "Osmosis"

Reply:
revert back from "terra" to "Osmosis"